src/test/java/org/minions/devfund/abel/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,122 @@
+package org.minions.devfund.abel.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Class for Movie Database Unit Tests.
+ */
+public class MovieDatabaseTest {
+
+    private static final String NEW_MOVIE = "NewMovie";
+    private static final String ACTOR1 = "Actor1";
+    private static final String ACTOR2 = "Actor2";
+    private static final String NEW_MOVIE2 = "NewMovie2";
```

Comment:
Don't create this constants.
Consider using final local variables inside test methods.